cmd/pbstatuser/main.go

```diff
@@ -144,7 +144,7 @@ func checkSourceAvailabilityAWS(ctx context.Context) {
 
 	for s := range chAws {
 		logger.Trace().Msgf("Checking AWS source availability status %s", s.SourceApplicationID)
-		metrics.ObserveAvailabilityCheckReqsDuration(models.ProviderTypeAWS, func() error {
+		metrics.ObserveAvailabilityCheckReqsDuration(models.ProviderTypeAWS.String(), func() error {
```

Comment:
Why did you add `.String()`?
it is not needed, it is present [here](https://github.com/RHEnVision/provisioning-backend/blob/bad43f81bf343b9f3898ab69ded82b6198804bd2/internal/metrics/metrics.go#L55)

`provisioning_source_availability_check_request_duration_ms_bucket{component="statuser",error="true",service="provisioning",type="aws",le="5000"} 2`